BoxContentSDK/BoxContentSDK/Requests/BOXFolderDeleteRequest.h

```diff
@@ -7,6 +7,18 @@
 
 @interface BOXFolderDeleteRequest : BOXRequestWithSharedLinkHeader
 
+// NOTE: Both the associateID and requestDirectoryPath values are required for performing the request in the background.
+/**
+ Caller provided unique ID to execute the request as a NSURLSession background task.
+ This is a required value for performing the request in the background.
+ */
+@property (nonatomic, readwrite, strong) NSString *associateId;
+
+/**
+ Caller provided directory path for the result payload of the background operation to be written to.
+ */
+@property (nonatomic, readwrite, copy) NSString *requestDirectoryPath;
```

Comment:
do we need this property for delete operation given we won't expect response from the request?

Reply:
it might be ok to keep this property to be consistent with all background tasks. We might not parse the content of the response in this path, but as a background task, we can expect the property to exist.

Reply:
sounds good.